### ISSUE TITLE
The source directory [directory] does not contain a CMakeLists.txt file.

### ISSUE THREAD

**Issue by RustyHorns** [Bionus/imgbrd-grabber#2820]
**Steps to reproduce**

1. Follow compiling instructions for Linux
2. Do `./build.sh`

**Expected behavior**

It build everything as explained in the documentation and I can go to the next step

**Context**

Running ./build.sh return this
```
CMake Error at lib/CMakeLists.txt:13 (add_subdirectory):
  The source directory

    /home/rusty_horns/Programs/imgbrd-grabber-7.9.1/src/lib/vendor/lexbor

  does not contain a CMakeLists.txt file.


CMake Error at gui-qml/languages/CMakeLists.txt:14 (qt_create_translation):
  Unknown CMake command "qt_create_translation".


-- Configuring incomplete, errors occurred!
```

**Screenshots**

![Screenshot from 2022-10-17 22-18-40](https://user-images.githubusercontent.com/87550628/196275126-2a87f6e8-9c86-4c45-aee3-0eeda02e1029.png)

**System information**

- OS: [Ubuntu 20.04.5 LTS]
- Grabber version: [Version 7.9.1]

**Additional context**

This is my first time making one of these, I'm sorry if I'm unclear or missed something.
I saw another topic touching on one of those errors, but the solution found didn't help in my case. (using `qmake --version` only show me this; 
![Screenshot from 2022-10-17 22-31-30](https://user-images.githubusercontent.com/87550628/196277354-ce6607d6-b25b-4a69-8c24-2a3f2b06838e.png) )

**output log**
[CMakeOutput.log](https://github.com/Bionus/imgbrd-grabber/files/9804312/CMakeOutput.log)

**Comment by Bionus** [comment]
Did you clone the repository recursively, including sub-modules such as Lexbor which seem to be missing?
You can do so by running `git submodule update --init --recursive`.

**Comment by RustyHorns** [comment]
I thought I did, but apparently not.
But running `git submodule update --init --recursive.` do nothing, and I still get the same errors

**Comment by RustyHorns** [comment]
I reinstalled everything from scratch and this time the issue with the missing CMakeLists.txt is gone, but I still have this

```
CMake Error at gui-qml/languages/CMakeLists.txt:14 (qt_create_translation):
  Unknown CMake command "qt_create_translation".
```
And I still can't find how to fix it, I looked at other similar issues but I don't understand what I'm supposed to do.

**Comment by Bionus** [comment]
Make sure you installed the proper Qt version and dev libraries. Right now, Grabber is targeting Qt 5.15.2.
On platforms other than Windows, I recommend [aqtinstall](https://github.com/miurahr/aqtinstall) to install Qt.

If you're confident you installed the proper dependencies, you can also ignore the `build.sh`  file and just do:
```
mkdir build
cd build
cmake ../src
make -j8
```

On a more general question and by curiosity, any reason you're building from source instead of using the provided AppImage?

**Comment by RustyHorns** [comment]
I gave up and installed windows on the pc because I was getting fed up of staying stuck like that, so I don't have any screenshots or logs on hand, but the installed version of qt was the latest from their website, even if I couldn't get it to use anything else than 5.12.8 for whatever reason (I couldn't find how to correct that even after asking in other places).

And I used the source because trying to install the appimage also didn't work at all, I just got errors and it wouldn't install anything.

On a side note, it was still a headache to use on windows because it keep crashing if I try to use it on the same computer it's storing the images on, and after trying on three separate computers I found out having it run in one and use another's shared drive works fine. I can't figure out why and at this point I'm just glad it works at all
